google-cloud-storage/src/main/java/com/google/cloud/storage/StorageClass.java

```diff
@@ -41,22 +41,42 @@ public StorageClass apply(String constant) {
   private static final StringEnumType<StorageClass> type =
       new StringEnumType(StorageClass.class, CONSTRUCTOR);
 
-  /** Regional storage class. */
-  public static final StorageClass REGIONAL = type.createAndRegister("REGIONAL");
-
-  /** Multi-regional storage class. */
-  public static final StorageClass MULTI_REGIONAL = type.createAndRegister("MULTI_REGIONAL");
+  /**
+   * Standard storage class. See: https://cloud.google.com/storage/docs/storage-classes for details
+   */
+  public static final StorageClass STANDARD = type.createAndRegister("STANDARD");
 
-  /** Nearline storage class. */
+  /**
+   * Nearline storage class. See: https://cloud.google.com/storage/docs/storage-classes for details
+   */
   public static final StorageClass NEARLINE = type.createAndRegister("NEARLINE");
 
-  /** Coldline storage class. */
+  /**
+   * Coldline storage class. See: https://cloud.google.com/storage/docs/storage-classes for details
+   */
   public static final StorageClass COLDLINE = type.createAndRegister("COLDLINE");
 
-  /** Standard storage class. */
-  public static final StorageClass STANDARD = type.createAndRegister("STANDARD");
+  /**
+   * Archive storage class. See: https://cloud.google.com/storage/docs/storage-classes for details
+   */
+  public static final StorageClass ARCHIVE = type.createAndRegister("ARCHIVE");
+
+  /**
+   * Regional storage class (deprecated). See: https://cloud.google.com/storage/docs/storage-classes
+   * for details
+   */
+  public static final StorageClass REGIONAL = type.createAndRegister("REGIONAL");
 
-  /** Durable Reduced Availability (deprecated) */
+  /**
+   * Multi-regional storage class (deprecated).
+   * See:https://cloud.google.com/storage/docs/storage-classes for details
```

Comment:
please add a space after the : (same below too)

Reply:
done